src/V3EmitMk.cpp

```diff
@@ -24,18 +24,514 @@
 
 VL_DEFINE_DEBUG_FUNCTIONS;
 
+class OutputGroup final {
+public:
+    struct FileOrConcatenatedFilesList final {
+        std::string m_fileName;
+        std::vector<std::string> m_concatenatedFilenames{};
```

Comment:
```suggestion
        std::string m_filename;
        std::vector<std::string> m_concatenatedFilenames{};
```

And comment all member variables please.